server/sonar-web/src/main/js/apps/webhooks/components/WebhookActions.tsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import * as React from 'react';
+import CreateWebhookForm from './CreateWebhookForm';
+import ActionsDropdown, {
+  ActionsDropdownItem,
+  ActionsDropdownDivider
+} from '../../../components/controls/ActionsDropdown';
+import ConfirmButton from '../../../components/controls/ConfirmButton';
+import { translate, translateWithParameters } from '../../../helpers/l10n';
+import { Webhook } from '../../../app/types';
+
+interface Props {
+  onDelete: (webhook: string) => Promise<void>;
+  onUpdate: (data: { webhook: string; name: string; url: string }) => Promise<void>;
+  webhook: Webhook;
+}
+
+interface State {
+  updating: boolean;
+}
+
+export default class WebhookActions extends React.PureComponent<Props, State> {
+  mounted: boolean = false;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Type boolean trivially inferred from a boolean literal, remove type annotation ([typescript:S3257](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3257))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2FMMF-647%2FSONAR-10345_manage_webhooks&issues=AWGT1rXcQeDOAQ7I7jpO&open=AWGT1rXcQeDOAQ7I7jpO)

[ISSUE_KEY]: AWGT1rXcQeDOAQ7I7jpO